Travis failure

https://travis-ci.org/InstitutMaupertuis/fanuc_grinding/builds/136085880
```
[fanuc_r1000ia_sls_2_grinding_resources:make] /home/travis/ros/ws_fanuc_grinding/src/fanuc_grinding/institut_maupertuis_robots_descriptions/fanuc_r1000ia_grinding/fanuc_r1000ia_sls_2_grinding_resources/src/test_calibration.cpp:11:33: fatal error: pcl_ros/point_cloud.h: No such file or directory
[fanuc_r1000ia_sls_2_grinding_resources:make]  #include <pcl_ros/point_cloud.h>
[fanuc_r1000ia_sls_2_grinding_resources:make]                                  ^
[fanuc_r1000ia_sls_2_grinding_resources:make] compilation terminated.
```

`institut_maupertuis_robot_descriptions` compiles fine on Travis:
https://travis-ci.org/InstitutMaupertuis/institut_maupertuis_robots_descriptions

--- a/.travis.yml
+++ b/.travis.yml
@@ -27,6 +27,7 @@ install:
   - git clone https://github.com/InstitutMaupertuis/institut_maupertuis_robots_descriptions.git
   - git clone https://github.com/InstitutMaupertuis/bezier.git
   - git clone https://github.com/InstitutMaupertuis/fanuc_post_processor.git
+  - git clone https://github.com/davetcoleman/rviz_visual_tools.git
   - git clone https://github.com/InstitutMaupertuis/advanced_industrial_ci.git .advanced_ci_config
   - git clone https://github.com/ros-industrial/industrial_ci.git .ci_config
 

--- a/publish_meshfile/CMakeLists.txt
+++ b/publish_meshfile/CMakeLists.txt
@@ -3,6 +3,7 @@ project(fanuc_grinding_publish_meshfile)
 
 find_package(catkin REQUIRED COMPONENTS
              genmsg
+             pcl_ros
              roscpp
              std_msgs
              visualization_msgs)

--- a/publish_meshfile/package.xml
+++ b/publish_meshfile/package.xml
@@ -10,6 +10,7 @@
 
   <build_depend>genmsg</build_depend>
   <build_depend>message_generation</build_depend>
+  <build_depend>pcl_ros</build_depend>
   <build_depend>roscpp</build_depend>
   <build_depend>std_msgs</build_depend>
   <build_depend>visualization_msgs</build_depend>
@@ -17,6 +18,7 @@
   <build_export_depend>std_msgs</build_export_depend>
 
   <exec_depend>message_runtime</exec_depend>
+  <exec_depend>pcl_ros</exec_depend>
   <exec_depend>roscpp</exec_depend>
   <exec_depend>std_msgs</exec_depend>
   <exec_depend>visualization_msgs</exec_depend>

